From e98978f3dabfac76e65bdc346cbc9fb3023bc8c7 Mon Sep 17 00:00:00 2001
From: Ajay Thomas <ajay.thomas.david.rajamanickam@intel.com>
Date: Mon, 13 Jul 2015 19:47:46 +0530
Subject: [PATCH] EM/OPREGION: Provide suport for V1P2A VR

This patch provides support for V1P2A VR
in the whiskey cove opregion driver
for cherrytrail.

Change-Id: If0dd69c8f6dfc1b0528ca081ed08ab2ee794ebbd
Tracked-On: https://jira01.devtools.intel.com/browse/GMINL-13196
Signed-off-by: Ajay Thomas <ajay.thomas.david.rajamanickam@intel.com>
---
 drivers/acpi/pmic_opregion/intel_soc_pmic_wc_opregion.c | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/drivers/acpi/pmic_opregion/intel_soc_pmic_wc_opregion.c b/drivers/acpi/pmic_opregion/intel_soc_pmic_wc_opregion.c
index 79f6e2c2a4b8..834f1d3b8e17 100644
--- a/drivers/acpi/pmic_opregion/intel_soc_pmic_wc_opregion.c
+++ b/drivers/acpi/pmic_opregion/intel_soc_pmic_wc_opregion.c
@@ -43,6 +43,13 @@ static struct pmic_pwr_table pwr_table[] = {
 		},
 	}, /* V18X -> V1P8SX */
 	{
+		.address = 0x0c,
+		.pwr_reg = {
+			.reg = 0x59,
+			.bit = VR_MODE_NORMAL,
+		},
+	}, /* V12A -> V1P2ACNT */
+	{
 		.address = 0x10,
 		.pwr_reg = {
 			.reg = 0x5a,
-- 
1.9.1

